index.html

```diff
@@ -111,7 +111,7 @@ <h1 class="title toc-ignore">DT: An R interface to the DataTables library</h1>
 <p>You may install the stable version from CRAN, or the development version using <code>remotes::install_github('rstudio/DT')</code> if necessary (this website reflects the development version of <strong>DT</strong>):</p>
 <pre class="sourceCode r"><code class="sourceCode r"><span class="co"># if (!require(&quot;DT&quot;)) install.packages(&#39;DT&#39;)</span>
 xfun<span class="op">::</span><span class="kw">session_info</span>(<span class="st">&#39;DT&#39;</span>)</code></pre>
-<pre><code>## R version 3.6.1 (2019-07-05)
+<pre><code>## R version 3.5.3 (2019-03-11)
```

Comment:
Perhaps you need to update your R version at some point.

Reply:
🤪